cvat-core/src/session.js

```diff
@@ -565,6 +565,7 @@
                 start_frame: undefined,
                 stop_frame: undefined,
                 sequence_name: undefined,
+                version: undefined,
```

Comment:
what does this version refer to? software version, index of annotator, revision... ?
`version` looks like a too generic/overloaded word to use here, not sure what a better word to use, but at least think about it

Reply:
It refers to the index of annotator (but it's a bit more complicated, as one version gets annotated by multiple annotators if the task has thousands of frames).

I agree that version is too generic and I don't like using this word either. I've considered using other words for this term (variants, alternatives, sources, ~options~). But Sveta and David refer to it as _versions_, so I think there would be a lot of confusion if I used another term in the code.

Moreover, there is a ticket for VCS-like versions (https://p.daedalean.ai/T5144). I think that using term `revision` would be the right choice there.